lib/scenic/adapters/postgres/views.rb

```diff
@@ -54,6 +54,11 @@ def to_scenic_view(result)
             materialized: result["kind"] == "m",
           )
         end
+
+        def pg_identifier(name)
+          return name if name =~ /^[a-zA-Z_][a-zA-Z0-9_]*$/
```

Comment:
Removing this line caused double quoting in in-memory representations of the names (failing tests) and in the SQL dump (failing more tests, and breaking things). I wasn't able to fix that in ~an hour, so I reverted my attempts.